server/plugins/openAiWhisperPlugin.js

```diff
@@ -171,12 +172,15 @@ class OpenAIWhisperPlugin extends ModelPlugin {
     // Execute the request to the OpenAI Whisper API
     async execute(text, parameters, prompt, cortexRequest) {
         const { pathwayResolver } = cortexRequest;
+
         const { responseFormat, wordTimestamped, highlightWords, maxLineWidth, maxLineCount, maxWordsPerLine } = parameters;
-        cortexRequest.url = this.requestUrl(text);
+        // cortexRequest.url = this.requestUrl(text);
```

Comment:
Why commented out?  If it's not necessary we can remove?  Why is it not necessary?